src/debugger/Debugger.js

```diff
@@ -11,31 +11,28 @@
 
 import type { BabelNode } from "babel-types";
 import { Breakpoint } from "./Breakpoint.js";
-import { ExecutionContext } from "../realm.js";
 import invariant from "../invariant.js";
-import fs from "fs";
+import { DebugChannel } from "../DebugChannel.js";
 
 export class Debugger {
-  constructor(dbgFileLines: Array<string>) {
+  constructor(dbgFileLines: Array<string>, channel: DebugChannel) {
     this.breakpoints = new Map();
     this.prevBreakLine = 0;
     this.prevBreakCol = 0;
     this.parseCommands(dbgFileLines);
+    this.channel = channel;
   }
-  breakpoints: Map<number, Breakpoint>;
+  breakpoints: { [number]: Breakpoint };
```

Comment:
Why not use an array here? Also, why not provide for column breakpoints?  Another thing to consider is that there may be multiple files. We also need to discuss what to do about source maps.